core/src/main/java/org/kiwix/kiwixmobile/core/reader/ZimFileReader.kt

```diff
@@ -247,3 +254,5 @@ private val String.mimeType: String?
   get() = MimeTypeMap.getSingleton().getMimeTypeFromExtension(
     MimeTypeMap.getFileExtensionFromUrl(this)
   )
+
+data class NextSuggestion(val title: String, val url: String)
```

Comment:
This isn't a `NextSuggestion` merely a `Suggestion` though `SearchSuggestion` probably adds more context so that is better. 
This data class can be in its own class, doesn't feel like it belongs here anyways. Also use this class

Reply:
Thanks for pointing out the data class, I'd forgotten I had created it previously